pom.xml

```diff
@@ -218,7 +218,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.4</version>
+            <version>2.1.6</version>
```

Comment:
Keeping dependency version aligned with highest version in transitive dependencies